### PR TITLE
[logstash] add missing queue.capacity mappings

### PR DESCRIPTION
**Pull request by klacabane**
### Summary

Part of https://github.com/elastic/beats/issues/38164

### CODE REVIEW

File: packages/logstash/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.4.2"
+  changes:
+    - description: Add missing queue.capacity.* mappings
+      type: bugfix
+      link: tbd
```
Comment (miltonhultgren) [inline packages/logstash/changelog.yml:6]:
```suggestion
      link: https://github.com/elastic/integrations/pull/9339
```

**Review by miltonhultgren — APPROVED** [review]

**Comment by klacabane** [comment]
Agent is apparently unable to reach logstash service during system tests for the `node` and `node_stats` data streams. I can't reproduce locally. I know elastic-package introduced improvements on failure detection during tests recently and wondering if this could be a false positive triggered by that new mechanism (ie if there are any erroneous documents collected during tests we fail)

```
one or more errors found in documents stored in metrics-logstash.stack_monitoring.node-ep data stream: [0] found error.message in event: error making http request: Get "http://elastic-package-service-logstash-1:9600/": dial tcp 172.18.0.5:9600: connect: connection refused
```

**Comment by klacabane** [comment]
I think the previous errors about service being unavailable were fixed by https://github.com/elastic/integrations/pull/9339/commits/9e23173fd85d7b26e0feb31f905f6c50bdc89fdf but now getting errors about log data stream. Unfortunately I can't reproduce locally

```
one or more errors found in documents stored in logs-logstash.log-ep data stream: [0] found error.message in event: Duplicate field 'message'\n at [Source: (org.elasticsearch.common.io.stream.ByteBufferStreamInput); line: 1, column: 206]
[1] found error.message in event: Duplicate field 'message'\n at [Source: (org.elasticsearch.common.io.stream.ByteBufferStreamInput); line: 1, column: 209]
[2] found error.message in event: Duplicate field 'message'\n at [Source: (org.elasticsearch.common.io.stream.ByteBufferStreamInput); line: 1, column: 331]
```

**Comment by klacabane** [comment]
Lot of flakiness involved in the package system tests. Merging the green build but I'll log an issue to investigate the errors surfaced in the previous builds

**Pull request merged**